h/services/group_members.py

```diff
@@ -24,6 +24,29 @@ def __init__(self, db, user_fetcher, publish):
         self.user_fetcher = user_fetcher
         self.publish = publish
 
+    def get(self, group: Group, roles: list[GroupMembershipRoles] | None = None):
```

Comment:
I made the same comment in a similar PR, this might be better named `get_memberships` or similar, small extra typing but much better greppeability.

Reply:
Yep, renamed this to `get_memberships()` so we now have `get_membership(group, user)` to get a single membership and `get_memberships(group, roles=None)` to get all of a group's memberships.